src/MessageReader.js

```diff
@@ -4,15 +4,37 @@
 // found in the LICENSE file in the root directory of this source tree.
 // You may not use this file except in compliance with the License.
 
+// @flow
+
+import Buffer from "buffer/";
```

Comment:
Add a comment here why the trailing slash is needed?

Reply:
Why do we want to use the `@feross/buffer` implementation in node?

Reply:
I don't think it matters too much except that Flow needs to be able to typecheck Buffer. In reality, I think I added this when I was using TypeScript and it might not be necessary anymore; I'll try removing it.